src/services/SyncService.js

```diff
@@ -242,7 +242,7 @@ class SyncService {
 	async save({ force = false, manualSave = true } = {}) {
```

Comment:
We may still want to trigger the autosave somehow regular. I think this is no longer the case as only sync is called in an interval. Might be something to check with @max-nextcloud next week on how to best achieve that.

Reply:
It's actually still being triggered regularly. At least yesterday it was :see_no_evil:

Reply:
And that's the 30s autosave: https://github.com/nextcloud/text/blob/0d088d669fad3861387b27ca4e1b0cc51c147cab/src/services/SyncService.js#L88